<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankingSurvival</class>
 <widget class="QDialog" name="RankingSurvival">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="blood_splash">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>-20</y>
     <width>211</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets_ending.qrc">:/imgs/blood_splash.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_infcted_killed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Infected killed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_bullets_shot">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bullets shot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="blood_hand">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets_ending.qrc">:/imgs/blood_hand.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="amount_infcted_killed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>120</y>
     <width>331</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="blood_splash_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets_ending.qrc">:/imgs/blood_stain.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bullets_shot">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ranking">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>40</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ranking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gameplay_duration">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gameplay Duration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="your_stats">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>50</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your Stats</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameplay_duration">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>20</width>
     <height>621</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="blood_splash_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>560</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets_ending.qrc">:/imgs/blood_stain.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>1081</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets_ending.qrc">:/imgs/background_40-min.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>blood_splash</zorder>
  <zorder>label_infcted_killed</zorder>
  <zorder>label_bullets_shot</zorder>
  <zorder>blood_hand</zorder>
  <zorder>amount_infcted_killed</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>blood_splash_2</zorder>
  <zorder>bullets_shot</zorder>
  <zorder>ranking</zorder>
  <zorder>label_gameplay_duration</zorder>
  <zorder>your_stats</zorder>
  <zorder>gameplay_duration</zorder>
  <zorder>line</zorder>
  <zorder>blood_splash_3</zorder>
 </widget>
 <resources>
  <include location="assets_ending.qrc"/>
 </resources>
 <connections/>
</ui>
